plugins/slack/app/models/slack_notification.rb

```diff
@@ -4,18 +4,20 @@ def initialize(deploy)
     @stage = deploy.stage
     @project = @stage.project
     @user = @deploy.user
+    @webhook = @stage.slack_webhooks.first
```

Comment:
@steved Yes. This was written as a one-to-many but it's being as a one-to-one. So I just stuck to whatever was present already.